docs/DeveloperGuide.md

```diff
@@ -517,9 +517,12 @@ Use case ends.
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
-* **Claims**: A formal request by a client for reimbursement for losses that are covered by specific insurance plans.
+* **Client**: A person who purchased at least one insurance plan from the insurance agent.
```

Comment:
What about potential customers? We might still consider them as "clients" even though they have not yet purchased anything

Reply:
What about

Client: A customer that the user is speaking to. Has either purchased a plan, or is keen to do so

Reply:
Nice, thanks! Will make the edits